packages/wdio-config/tests/node/configparser.test.ts

```diff
@@ -689,6 +689,35 @@ describe('ConfigParser', () => {
 
             expect(configParser.getCapabilities()).toMatchObject([{ browserName: 'safari' }])
         })
+
+        it('should allow to specifying services', async () => {
+            const configParser = await ConfigParserForTest(FIXTURES_CONF)
+            await configParser.initialize({
+                services: [
+                    ['MyService', { timestamp: 1709633731716 }]
```

Comment:
Can we add a class here too? Let's make sure that the use case from the original issue is covered here.

Reply:
Updated this with class.